src/styles/index.scss

```diff
@@ -0,0 +1,18 @@
+@import './variables';
+@import './header';
+@import './main';
+@import './card';
+@import './stars';
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Specify tags. For example:
```
h1, h2, p {
  margin: 0;
  padding: 0;
}
```
![image](https://github.com/mate-academy/layout_catalog/assets/54936044/a69c3738-479a-4f93-97d4-986e87ce8e16)